README.md

```diff
@@ -67,12 +67,31 @@ Be sure to install [VS Code](https://code.visualstudio.com/Download) and create
 
 ![Run selection query](media/selection-query.gif)
 
+### 4. Upload GraphQL Schema
+
+1. Open a `.graphql` file containing your GraphQL schema as per the [FaunaDB specification](https://docs.fauna.com/fauna/current/api/graphql).
```

Comment:
I would expect the file suffix to be `.gql`
What is the standard here?

Reply:
I think both extensions are fine. I have used `.graphql` in my code base most of the times, but I think vscode understands `.gql` as well. Can include both options in the README.